addon/components/node-tree/tree/index.js

```diff
@@ -54,7 +57,9 @@ export default class NodeTreeComponent extends Component {
   @action
   async handleSelection(node) {
     const parent = await node[this.parentNodeName];
-    const nodeSelectedInitialState = node.isSelected;
+    const nodeSelectedInitialState = this.selectedNodes.has(node)
+      ? true
+      : false;
```

Comment:
I believe array's `has` should already return a boolean here, you might want to skip 61,62 lines.